[Magic Firewall] Added About and updated landing page (#2980)
cloudflare/cloudflare-docs: products/magic-firewall/src/content/about/traffic-types.md

---
title: Traffic types
pcx-content-type: concept
---

# Traffic types

Magic Firewall enables you to allow or block traffic on a variety of packet characteristics, such as source and destination IP, source and destination port, protocol, packet length, and bit field match. 

> Magic Firewall supports layers three and four — network and transport — protocols including TCP, UDP, and ICMP for example. Any type of layer three or four protocols can go through Magic Firewall and then be matched on those protocols.

Reviewer: ```suggestion
Magic Firewall supports layers three and four — network and transport — protocols such as TCP, UDP, and ICMP. Any type of layer three or four protocols can go through Magic Firewall and then be matched on those protocols.
```

> To view the list of fields, refer to [Magic Firewall fields](/reference/magic-firewall-fields).

Reviewer: ```suggestion
To view the list of available fields, refer to [Magic Firewall fields](/reference/magic-firewall-fields).
```
